pages/linux/steghide.md

```diff
@@ -0,0 +1,28 @@
+# steghide
+
+> Steganography tool for JPEG, BMP, WAV and AU file formats.
+> More information: <https://github.com/StefanoDeVuono/steghide>.
+
+- Embed data in a PNG image:
+
+`steghide embed --coverfile {{path/to/image.png}} --embedfile {{path/to/data.txt}} --passphrase "{{passphrase}}"`
```

Comment:
Is there a way to prompt for the passphrase or have it stored in a file? This isn't particularly secure.